lib/testing_appserver/modules/documents/appserver_documents.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../../top_toolbar/appserver_top_toolbar'
+
+module TestingAppServer
+  # AppServer Documents module
+  # https://user-images.githubusercontent.com/40513035/141085330-09584b43-6a2c-419c-9414-0c01219ea5a7.png
+  class AppServerDocuments
```

Comment:
Actually since we got namespace `TestingAppServer` I think we do not need `AppServer` part in name of each class

maybe something like `class DocumentsModule`?
And same for all other cases